### ISSUE TITLE
What's the reason for using the same validation and test set for WFLW dataset

### ISSUE THREAD

**Issue by superkido511** [open-mmlab/mmpose#1896]
I'm trying to train a custom face keypoints detector and use WFLW pretrained model for finetuning. I notice that the training config file use the same data for booth validation and test set. Is there any specific reason for this ? By my understanding, this would make us unable to evaluate the ability to generalize of the trained model.
https://github.com/open-mmlab/mmpose/blob/master/configs/face/2d_kpt_sview_rgb_img/deeppose/wflw/res50_wflw_256x256_softwingloss.py#L112

**Comment by jin-s13** [comment]
Please note that wflw has many test subsets. 

        |   |── face_landmarks_wflw_test.json
        |   |── face_landmarks_wflw_test_blur.json
        |   |── face_landmarks_wflw_test_occlusion.json
        |   |── face_landmarks_wflw_test_expression.json
        |   |── face_landmarks_wflw_test_largepose.json
        |   |── face_landmarks_wflw_test_illumination.json
        |   |── face_landmarks_wflw_test_makeup.json

face_landmarks_wflw_test.json  is for validation, and others are for evaluation (generalization ability). All results are reported in mmpose.

**Comment by superkido511** [comment]
I see, thanks
